test/api/endpoints/lossClaim/list/v3/ListLossClaimsControllerSpec.scala

```diff
@@ -140,9 +136,25 @@ class ListLossClaimsControllerSpec
     """.stripMargin
   )
 
+  trait NonTysTest extends Test {
+    override def taxYear: String       = "2018-19"
+    val rawData: ListLossClaimsRawData = ListLossClaimsRawData(nino, Some(taxYear), Some(selfEmployment), Some(businessId), Some(claimType))
+
+    val request: ListLossClaimsRequest =
+      ListLossClaimsRequest(Nino(nino), Some(TaxYear("2019")), None, Some(businessId), Some(TypeOfClaim.`carry-sideways`))
+  }
+
+  trait TysTest extends Test {
```

Comment:
I don't think we need to haver controller tests both TYS and non-TYS as there is no logic in the controller that depends on the taxYear. 

Also, as Matt's updates have removed the need to add mapping of errors to the controller, I don't think this class needs touching at all

Reply:
See my comment on the ControllerSpec for Jeremy's PR https://github.com/hmrc/self-assessment-bsas-api/pull/371